webmails/nginx-webmail.conf

```diff
@@ -36,7 +36,7 @@ server {
         try_files $uri $uri/ /index.php$args;
     }
 
-    location ~ \.php$ {
+    location ~ ^(.+\.php)(.*)$ {
```

Comment:
Wouldn't it be better to use the same regex as in the line below?

I don't see why there are two capturing groups and a ^ and $ now. Just removing the $ should have the same effect. 

But I'd prefer to use the same regex.